### PR TITLE
Clean up

### PR DESCRIPTION
**Pull request by jacobthill**

### CODE REVIEW

File: catalogs/harvard.yaml [hunk 1]
```diff
@@ -4,23 +4,102 @@ metadata:
   schedule: "0 10 * * *"
 sources:
   ihp:
-    driver: oai_xml
+    driver: xml
```
Comment (aaron-collier) [inline catalogs/harvard.yaml:7]: Did we want to change this xml? I thought we discussed keeping it as oai_xml, but perhaps that didn't work.

**Review by aaron-collier — CHANGES_REQUESTED** [review]

**Comment by aaron-collier** [comment]
If someone merges this before me, please squash the commits.

File: catalogs/harvard.yaml [hunk 2]
```diff
@@ -6,28 +6,14 @@ sources:
   ihp:
     driver: oai_xml
     args:
-      collection_url: https://api.lib.harvard.edu/oai
-      metadata_prefix: oai_dc
-      set: "ihp"
+      collection_url: https://api.lib.harvard.edu/oai?verb=ListRecords&metadata_prefix=mods&set=ihp
```
Comment (aaron-collier) [inline catalogs/harvard.yaml:9]: The build failure you're seeing is due to moving the `metadata_prefix` up into the URL. This should be an argument (required by the driver), as well as the `set`

Apologies, I should have mentioned that would be necessary when using `oai_xml`

Comment (jacobthill) [reply catalogs/harvard.yaml:9]: Oops, too many changes. One minute

**Pull request merged**